src/components/Poster.tsx

```diff
@@ -0,0 +1,76 @@
+// src/components/Poster.tsx
+import React from 'react'
+import Image from 'next/image'
+import styles from './Poster.module.css'
+
+export type PosterProps = {
+  type: 'image' | 'gif' | 'video'
+  src: string
+  alt?: string            // for images/gifs
```

Comment:
I think we don't need it, because we will only display media, you will not be able to see text